src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -2215,140 +2078,134 @@ public class RemoveServiceCommand : ServiceBaseCommand
         protected override void ProcessRecord()
         {
             ServiceController service = null;
-            string serviceComputerName = null;
-            foreach (string computer in ComputerName)
+            bool objServiceShouldBeDisposed = false;
+            try
             {
-                bool objServiceShouldBeDisposed = false;
-                try
+                if (InputObject != null)
```

Comment:
This code looks very similar to line 1485. Can we refactor?

Reply:
They are from two different cmdlets: Set-Service (line 1485) and Remove-Service (2083)